test/test_helper.rb

```diff
@@ -6,13 +6,9 @@
 require 'minitest/autorun'
 require "minitest/reporters"
 require 'pathname'
-require 'pry'
+# require 'pry' # TODO: What was this used for?
```

Comment:
I think I was having trouble getting cucumber and debugging working from VSCode. And ended up using pry to inspect things... though, memory fails me...